labs/lab-1-html-and-css/style.css

```diff
@@ -0,0 +1,122 @@
+body{
+  background-color: #f9fafc;
+}
+
+h1{
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+}
+
+h3{
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+}
+
+p{
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+  color: #69696a;
+}
+
+nav a {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 800;
+  margin: 10px;
+  position: relative;
+}
+
+a:link {
+  color: #69696a;
+  text-decoration: none;
+}
+
+.top-header {
+  display: inline;
+}
+
+.navigation {
+  float: right;
+}
+
+.main-content {
+  margin-top: 100px;
+}
+
+.main-header {
+  margin: 20px;
+}
+
+.circles {
+  display: relative;
+}
+
+.main-paragraph {
+  border-left: 3px solid #69696a;
+  padding-left: 10px;
+  width: 50%;
+  margin: 20px;
+}
+
+.circle-red {
+  position: relative;
```

Comment:
I would expect to see all the circles using `position: absolute`, not `relative`; they're not slightly nudged from where they'd be otherwise, they're in a totally new position! So absolute is the way to go; and place all three *inside* a container div that has `position: relative`

That will help fix the fact that those circles drift right off the screen around tablet-width, 768px.